src/wasm-type.h

```diff
@@ -137,28 +142,32 @@ class Type {
   Type(Rtt);
 
   // Predicates
-  //                 Compound Concrete
-  //   Type        Basic │ Single│
-  // ╒═════════════╦═│═╤═│═╤═│═╤═│═╤═══════╕
-  // │ none        ║ x │   │   │   │       │
-  // │ unreachable ║ x │   │   │   │       │
-  // ├─────────────╫───┼───┼───┼───┤───────┤
-  // │ i32         ║ x │   │ x │ x │ I     │ ┐ Number
-  // │ i64         ║ x │   │ x │ x │ I     │ │  I_nteger
-  // │ f32         ║ x │   │ x │ x │   F   │ │  F_loat
-  // │ f64         ║ x │   │ x │ x │   F   │ │  V_ector
-  // │ v128        ║ x │   │ x │ x │     V │ ┘
-  // ├─ Aliases ───╫───┼───┼───┼───┤───────┤
-  // │ funcref     ║ x │   │ x │ x │ f  n  │ ┐ Ref
-  // │ anyref      ║ x │   │ x │ x │ f? n  │ │  f_unc
-  // │ eqref       ║ x │   │ x │ x │    n  │ │  n_ullable
-  // │ i31ref      ║ x │   │ x │ x │       │ │
-  // │ dataref     ║ x │   │ x │ x │       │ │
-  // ├─ Compound ──╫───┼───┼───┼───┤───────┤ │
-  // │ Ref         ║   │ x │ x │ x │ f? n? │◄┘
-  // │ Tuple       ║   │ x │   │ x │       │
-  // │ Rtt         ║   │ x │ x │ x │       │
-  // └─────────────╨───┴───┴───┴───┴───────┘
+  //                      Compound Concrete
+  //   Type             Basic │ Single│
+  // ╒══════════════════╦═│═╤═│═╤═│═╤═│═╤═══════╕
+  // │ none             ║ x │   │   │   │       │
```

Comment:
Nit: I know it's preexisting, but what do you think about changing `x` to `o` in this table? We know what this means, but `x` can mean a negative, so..

Reply:
I'd be fine either way. Could also be a checkmark ✓

Reply:
The whole box is already made of non-alphanumeric chars, so checkmark sgtm ✔️